include/definitions.h

```diff
@@ -8,7 +8,7 @@
  *  GPIO
  */
 //#define LED_BUILTIN (gpio_num_t)13
-#define GPIO_INTERRUPT_PIN 33 // 25
+#define GPIO_INTERRUPT_PIN 0 //33 // 25
```

Comment:
Is this the right pin number, or just a placeholder? Looking at the schematic, is should be number 33...